.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Please modify this file so you don't include neither the node modules nor the package-lock.json.

Your file can look like this:

```
node_modules
package-lock.json
```